CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+
+v0.68.0
+------------------------------
+*August 30, 2018*
+
+### Changed
+- Updated heading font size to match latest UI designs.
```

Comment:
Looks like a font-weight is being changed…

Reply:
Oh $*^%! Will update now 😄